images/api/src/Schema.js

```diff
@@ -175,37 +174,48 @@ class Schema {
 
 
     let answers = [];
-    await pg.select().table("answers").where({tableID: req.params.uuid}).then(function(a) {
+    await pg.select().table('answers').where({tableID: req.params.uuid}).then(function(a) {
       answers = a;
     })
 
 
     // @TODO: make sure param 'uuid' is of type uuid
     await pg.select()
-      .table("schema")
-      .where({"schema.uuid": req.params.uuid})
-      .leftJoin('cells', 'schema.id', "=", "cells.tableID")
-      .then( function (r) {
+      .table('schema')
+      .where({'schema.uuid': req.params.uuid})
+      .leftJoin('cells', 'schema.id', '=', 'cells.tableID')
+      .then(async function (r) {
         if(r.length > 0) {
 
-          result["uuid"] = req.params.uuid;
-          result["title"] = r[0].title;
+          result['uuid'] = req.params.uuid;
+          result['title'] = r[0].title;
+          result['consumer'] = r[0].consumer;
+          result['status'] = r[0].published;
+          result['publication'] = {
+            from: r[0].opens,
+            to: r[0].closes
+          };
           const temp = [];
           Object.keys(r[0].headers).map((key, index) => {
             temp.push(key);
           });
-          result["headers"] = temp;
+          result['headers'] = temp;
 
 
-          result["created_at"] = r[0].created_at;
-          result["created_at"] = r[0].created_at;
-          result["updated_at"] = r[0].updated_at;
+          result['created_at'] = r[0].created_at;
+          result['updated_at'] = r[0].updated_at;
 
+          result['roles'] = [];
```

Comment:
Right here, all roles are stored on a scheme, should be limlited to whatever the front-end sends, but that's not the case (yet)